REQUIRE

```diff
@@ -1,5 +1,4 @@
 julia 0.4.6
-JSON 0.8.0
+JSON 0.6.0
 MathProgBase 0.5.4
 JuMP 0.14.0
-Compat 0.9.3
```

Comment:
https://github.com/lanl-ansi/PowerModels.jl/blob/35ac71791b6dd86d3b6d55cf55c4d5cc2a573c54/src/PowerModels.jl#L8

the tests only pass because one of the particular versions of other dependencies that are getting used at the moment happen to depend on Compat - that's not guaranteed to always be the case